### DISCUSSION TITLE
Script fails to run, tells me to check my date

### DISCUSSION THREAD

**Discussion by steveharman** [Guilouz/Creality-Helper-Script-Wiki#455] [Issues]
I've had the script installed for months.  I went to run it today and I get an error saying Download Failed and to check my date & time.  

The printer's date and time seem to be identical to my Mac (and watch!) so I'm a little confused.

![k1max](https://github.com/Guilouz/Creality-K1-Series/assets/12557929/6400231a-5ce2-4066-b025-7530808ee9c5)

**Comment by Guilouz** [comment]
Read the wiki, old script has been removed

**Comment by steveharman** [comment]
Thanks!
